### PR TITLE
Upgraded circe, doobie, http4s, and scala

### PR DESCRIPTION
**Pull request by mpilquist**
@ChristopherDavenport Could you do an RC1 or M3 release after this is merged? Thanks!

### CODE REVIEW

File: modules/core/src/test/scala/io/chrisdavenport/fuuid/FUUIDArbitraries.scala [hunk 1]
```diff
@@ -9,8 +9,8 @@ trait FUUIDArbitraries {
   )
 
   implicit val arbFUUIDFunction: Arbitrary[FUUID => FUUID] = Arbitrary(
-    Arbitrary.arbitrary[FUUID].map(f1 => {f: FUUID => f1})
+    Arbitrary.arbitrary[FUUID].map(f1 => {_: FUUID => f1})
```
Comment (mpilquist) [inline modules/core/src/test/scala/io/chrisdavenport/fuuid/FUUIDArbitraries.scala]: Note this change was needed due to 2.12.7 being pickier about unused params.

**Comment by mpilquist** [comment]
Any chance of a release today?

**Comment by ChristopherDavenport** [comment]
On its way to central.

**Comment by mpilquist** [comment]
Thanks!

**Pull request merged**